homeassistant/components/trest_solar/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "trest_solar",
+  "name": "TrestSolarController",
+  "codeowners": ["@LordMax2"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/trest_solar",
+  "homekit": {},
+  "iot_class": "cloud_polling",
+  "requirements": ["trest_identity==1.0.0", "trest_solar==1.0.1"]
```

Comment:
Do they need loggers?

Reply:
They do not need loggers, I used two libs to create one for each layer of application, but of course the identity should be included in the solar one, because otherwise the solar one isnt going to work.